src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/translator_cache_dir_disabled.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$container->loadFromExtension('framework', [
+    'translator' => [
+        'cache_dir' => '',
```

Comment:
`'cache_dir' => null,`?

Reply:
Fixed:
I wanted to test the empty string.
what do you think about ?